src/p2p/net_node.inl

```diff
@@ -1404,33 +1491,32 @@ bool node_server<t_payload_net_handler>::make_new_connection_from_peerlist(
 
         if (use_white_list) {
             // if using the white list, we bias towards peers we've been using recently
-            random_index = get_random_exp_index(filtered.size());
-            std::lock_guard lock{m_used_stripe_peers_mutex};
-            if (next_needed_pruning_stripe > 0 &&
-                next_needed_pruning_stripe <= (1ul << cryptonote::PRUNING_LOG_STRIPES) &&
-                !m_used_stripe_peers[next_needed_pruning_stripe - 1].empty()) {
-                const auto na = m_used_stripe_peers[next_needed_pruning_stripe - 1].front();
-                m_used_stripe_peers[next_needed_pruning_stripe - 1].pop_front();
```

Comment:
This prioritises syncing nodes that are on the same pruning stripe as us. I believe this is done to make us have nodes with the relevant data we would need.

However I dont think we are getting a lot of pruning in the wild, plus if we select bad nodes because they happen to be randomly in the same batch as us then it might be causing our syncing issues.

But on the other hand we lose this syncing a pruned blockchain path.